pravega-sensor-collector/src/main/java/io/pravega/sensor/collector/rawfile/FileNameWithOffset.java

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.sensor.collector.rawfile;
+
+import java.util.Objects;
+
+/**
+ * File name and file size
+ */
+public class FileNameWithOffset implements Comparable<FileNameWithOffset> {
+    public final String fileName;
+    /**
+     * In some contexts, this is the size of the file.
+     * In the future, this will represent the offset in the file for incrementally ingesting growing log files.
+     * This is partially implemented today.
+     * TODO: Clarify usage of offset.
```

Comment:
what is this TODO?

Reply:
This was brought over from https://github.com/apoorva918/pravega-sensor-collector/blob/57f835d505c1d9f74b8e159f6a283fe2aa554c1a/pravega-sensor-collector/src/main/java/io/pravega/sensor/collector/file/FileNameWithOffset.java#L20C8-L20C8
I can remove it if its not required.